jsonschema-core.xml

```diff
@@ -2777,22 +2776,20 @@
                         </artwork>
                     </figure>
                     <t>
-                    Note that this pointer may not be resolvable by the normal JSON Pointer process
-                    due to the inclusion of these by-reference applicator keywords.
+                        Note that this pointer may not be resolvable by the normal JSON Pointer process
+                        due to the inclusion of these by-reference applicator keywords.
                     </t>
                     <t>
                         The JSON key for this information is "evaluationPath".
                     </t>
                 </section>
 
-                <section title="Keyword Absolute Location">
+                <section title="Subschema Absolute Location">
```

Comment:
Aren't we just making this "Schema Location"?  "absolute" and "relative" just ended up being confusing.

Reply:
The title isn't the property name.  It's supposed to be a bit more descriptive.  The other title has "relative" in it.

I can update them both, but I don't think just the humanized keyword name is sufficient here.

Reply:
Yeah my main concern is to drop the absolute/relative thing as I think it was more confusing than helpful, and we can work up better language in terms of dynamic vs static/lexical schema identification (not necessarily _those_ words, but better words around those concepts).

Because really, this organizes output by dynamic scope more than anything else.  The same [sub]schema might appear may times, but it's a different dynamic scope each time.